### PR TITLE
Updated Home Page Carrousel 

### PR DESCRIPTION
**Pull request by Claurt07**
Changed the entire code of the carrousel to improve stability, add autoslide, and change the look. 

**Changes:** 
- Css code to include code for the new Carrousel 
 - with 5 slides (intro, project, programs, tech update, and events 
- Homepage HTML with replaced code
- New JS file with carrousel code 
- Updated home page stats 
other
- Minor fix in Program page to remove duplicate countries

**Screenshots of the change:** 
image
image

### CODE REVIEW

**Comment by Claurt07** [comment]
@kshitijrajsharma this one is a bigger one man -- replaced a bigger chunk of code. Let me know what you think in general.

File: _layouts/home.html [hunk 1]
```diff
@@ -9,137 +9,157 @@
...
 
           <div class="container home-stats">
             <div class="home-stat">
-              <!-- <center><p class="loader"></p></center> -->
-              <!-- <p class="stat-number" id="Community-Mappers"></p> -->
-              <p class="stat-number">468K</p>
+              <p class="stat-number">491K</p>
               <p class="stat-title">Total Mappers<sup>&#9432;</sup>
-                <span class="tooltiptext">Total number of Mappers using <a href="https://tasks.hotosm.org/" target="_blank">HOT Tasking Manager</a>  - As of Dec. 2023</span>
+                <span class="tooltiptext">Total number of Mappers using <a href="https://tasks.hotosm.org/" target="_blank">HOT Tasking Manager</a>  - as of May 2024</span>
```
Comment (kshitijrajsharma) [inline _layouts/home.html:141]: We still manually update this ?

Comment (kshitijrajsharma) [reply _layouts/home.html:141]: For future : ohsomenow provides and API endpoint from which you can query and display the latest numbers

File: js/carrousel-homepage.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+  document.addEventListener('DOMContentLoaded', () => {
```
Comment (kshitijrajsharma) [inline js/carrousel-homepage.js:1]: Do you have source for it ? or its a plain code you have written ? if there is a source : You should mention it in top of the page !

**Review by kshitijrajsharma — APPROVED** [review]

**Comment by kshitijrajsharma** [comment]
Changes looks good to me , Merging the PR . You can address the comments later ( not that they are much specific haha )

**Pull request merged**